packages/fluentui/react-theming/package.json

```diff
@@ -29,7 +29,7 @@
     "lint:old": "tslint -t stylish -p .",
     "start-test": "just-scripts test:watch",
     "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest:snapshots"
```

Comment:
I think these scripts should get gone - we don't need a jest -u in these I think... otherwise, we should make sure these packages work with `just-scripts jest:snapshots`...

Reply:
@kenotron gotcha, thanks for the heads-up. I hadn't been tracking the repo convergence work so this was just to unblock myself on another PR. Should I just remove line 32 altogether?

Reply:
Yeah that's what I mean. I hope this helps :)

Reply:
Chatted offline; agreed to remove these lines.